webdriver/tests/bidi/browsing_context/locate_nodes/invalid.py

```diff
@@ -218,3 +218,24 @@ async def test_params_start_nodes_dom_node_not_element(
             locator={"type": "css", "value": "div"},
             start_nodes=[remote_reference],
         )
+
+
+@pytest.mark.parametrize("domain", ["", "alt"], ids=["same_origin", "cross_origin"])
+@pytest.mark.asyncio
+async def test_locate_by_context_invalid_context(bidi_session, inline, top_context, domain):
+    iframe_url_2 = inline("<div>foo</div>", domain=domain)
+    iframe_url_1 = inline(f"<div><iframe src='{iframe_url_2}'></iframe></div>", domain=domain)
```

Comment:
Please use the `iframe` fixture in those two lines. I assume we don't have to put it inside a div as well, right?